UnityProject/Assets/Scripts/Weapons/Projectiles/BulletEnergyShot.cs

```diff
@@ -0,0 +1,8 @@
+﻿public class BulletEnergyShot : BulletBehaviour
+{
+    public override void OnShoot()
+    {
+        damageType = DamageType.BURN; // sets damage hook to burn damage these are energy weapons
+        //Bullet specific stuff
```

Comment:
Why the "bullet specific stuff" explain what need to be added and use //FIXME or //TODO as a prefix ;)